### ISSUE TITLE
UI/UX Old and Stale

### ISSUE THREAD

**Issue by pjones235** [ActiveForums/ActiveForums#425]
I think the overall design of the ActiveForums module is old, stale, and in need of a redesign. Moreso for the Control Panel than the forums themselves. I know this isn't a technical issue, but I think the current UI/UX for admins isn't as good as it could be. I'd be happy to work on redesigning the vanilla theme. I think it would be really helpful if the admin UI was mobile responsive as well. I can create a pull request if people would like.

**Comment by Joezyz** [comment]
Hello everyone,
My wife, who uses this module, and has a background in software product management and UI wants to help with the re-design of the UI.

She was saying the same thing as the OP to me.  I read this post and asked her if she wanted to be involved.


Let me know if you would like her help.

-Joe

**Comment by pjones235** [comment]
Maybe if she wanted to provide a redesign and then myself/anyone else can go off of what she provides? I'd hate to spend too much time if there isn't enough interest.

**Comment by johnhenley** [comment]
Personally, I think the UI in the control panel is “good enough”. Since it’s only used by admins not end users, it should not be a high priority compared to many other many outstanding issues and improvements that could be made that would benefit all users.

John

On Jul 18, 2018, at 11:05 AM, Philip A. Jones <notifications@github.com<mailto:notifications@github.com>> wrote:


Maybe if she wanted to provide a redesign and then myself/anyone else can go off of what she provides? I'd hate to spend too much time if there isn't enough interest.

**Comment by Cambridgeport90** [comment]
I would agree with the "good enough" idea. Admins, the UI should be as simple as possible, not as flashy as possible.
